lib/core/managementClient.ts

```diff
@@ -23,6 +24,28 @@ export enum DispositionStatus {
   renewed = "renewed"
 }
 
+export interface RuleDescription {
+  filter?: SQLExpression | CorrelationFilter;
+  action?: SQLExpression;
+  name: string;
+}
+
+export interface SQLExpression {
+  expression: string;
+}
+
+export interface CorrelationFilter {
+  "correlation-id"?: string;
+  "message-id"?: string;
+  to?: string;
+  "reply-to"?: string;
+  label?: string;
+  "session-id"?: string;
+  "reply-to-session-id"?: string;
+  "content-type"?: string;
+  properties?: any;
+}
```

Comment:
In .Net, the above properties use Pascal Casing. I retained the same names as expected by AMQP. This is up for discussion though